packages/sysutils/systemd/scripts/module-overlays-setup

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+################################################################################
+#      This file is part of LibreELEC - http://www.libreelec.tv
+#      Copyright (C) 2017 Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+SYSTEM_OVERLAYS_DIR=/usr/lib/module-overlays
+OVERLAY_CONFIG_DIR=/storage/.cache/module-overlays
+KVER=$(uname -r)
+MODULES_DIR="/var/lib/modules/${KVER}"
+
+mkdir -p "${MODULES_DIR}"
+mkdir -p "${OVERLAY_CONFIG_DIR}"
+
+log() {
+  echo "module-overlays-setup: $@" > /dev/kmsg
+}
+
+apply_overlay() {
+ case "$1" in
```

Comment:
Substring expansion is a feature of bash, it's not in standard bourne shells.

In general I'd like to avoid bashism in shell scripts to keep them compatibe.

Also the script doesn't explicitly use bash (#!/bin/sh at start) so can't rely on having a bash so I used the slightly longer but compatible case approach.

But the missing space was a good catch!